services/src/main/java/org/exoplatform/onlyoffice/Config.java

```diff
@@ -473,10 +473,6 @@ public Config build() {
       /** The folder. */
       protected final String folder;  // 'Example Files'
 
-      // TODO there is also sharingSettings array where we can put users with
-      // different access/edit
-      // permissions: 'Full Access', 'Read Only'
```

Comment:
This information could be valuable to implement advanced sharing models. If removed here - note it somewhere to keep in account.

Reply:
Agreed.